src/HassClient/Model/ConnectionStatus.cs

```diff
@@ -0,0 +1,24 @@
+namespace JoySoftware.HomeAssistant.Model;
+
+/// <summary>
+///     Status of the connection to Home Assistant
+/// </summary>
+public enum ConnectionStatus
```

Comment:
These look like two separate states actually
- Ha is running or not
- Websocket is connected or not

Maybe they should be two bools, or just make the names a bit clearer.

Reply:
I want it to be "one state" so you can subscribe to it. I am thinking we use this in  ND to handle app lifecycle depending on state. I want to be able to log "Waiting for Home Assistant to finish starting up" or smth.

Reply:
I fixed the states to be more clear it is the status of the connection of HassClient. You can see if this is better now.